ui/app/components/clients/running-total.hbs

```diff
@@ -36,20 +32,15 @@
     </:stats>
 
     <:chart>
-      <Clients::Charts::VerticalBarGrouped
-        @data={{this.runningTotalData}}
-        @legend={{this.chartLegend}}
-        @upgradeData={{@upgradeData}}
-        @chartHeight="250"
+      {{! TODO SLW switching to basic from group loses the upgrade data info on the tooltip, confirm whether this is acceptable and revert if not }}
+      <Clients::Charts::VerticalBarBasic
         @chartTitle="Vault client counts"
+        @data={{this.runningTotalData}}
+        @dataKey="new_clients"
+        @chartHeight={{200}}
       />
     </:chart>
 
-    <:legend>
```

Comment:
I think we should probably keep this so users don't have to hover over the bars and read "new clients" via the tooltip (which isn't very accessible) 

Plus this gap without it looks a little spacious 
<img width="1078" alt="Screenshot 2025-05-12 at 3 33 01 PM" src="https://github.com/user-attachments/assets/697972a5-e438-402f-891b-ac85c27449dd" />